kite-data/kite-data-core/src/main/java/org/kitesdk/data/DatasetDescriptor.java

```diff
@@ -486,7 +487,7 @@ public Builder schemaLiteral(String s) {
      * @since 0.2.0
      */
     public <T> Builder schema(Class<T> type) {
-      this.schema = ReflectData.get().getSchema(type);
+      this.schema = DataModelUtil.getTypeSchema(type);
```

Comment:
I agree this is the right thing to do, but does this break compatibility? The fact that it causes tests to fail I think means yes.